src/components/Blog/Blog.react.js

```diff
@@ -206,6 +206,16 @@ class Blog extends Component {
     this.scrollToTop();
   };
 
+  showScrollToTop = () => {
+    let topButton = document.getElementById('topButton');
+    let y = window.scrollY;
+    if (y >= 90) {
```

Comment:
What is the logic for 90?

Reply:
@akshatnitd The logic for y is I felt after scrolling 90px down the top button should we visible.

Reply:
Okay, 90px looks good. But, please rename the variable `y` to a better name.